core/src/main/resources/jenkins/agents/CloudSet/_new.jelly

```diff
@@ -0,0 +1,48 @@
+<!--
+The MIT License
+
+Copyright (c) 2023, CloudBees Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<!--
+  2nd page in the "new cloud" page for displaying the entire configuration entries for the selected cloud type.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <l:layout permission="${app.ADMINISTER}" type="one-column" title="${%New cloud}">
+    <l:breadcrumb title="${%New cloud}"/>
+    <l:main-panel>
+      <f:form method="post" action="doCreate" name="config" class="jenkins-form">
+        <h1>${%New cloud}</h1>
+        <!-- main body of the configuration -->
+        <st:include it="${requestScope.instance}" class="${requestScope.descriptor.clazz}" page="config.jelly" />
```

Comment:
This breaks a feature of CloudBees CI:

```
jenkins/agents/CloudSet/_new.jelly:37:114: <st:include> No page found 'config.jelly' for class com.cloudbees.opscenter.client.cloud.CloudImpl
```

That cloud has other issues (it hard-codes a `name`) but perhaps at least
```suggestion
        <st:include it="${requestScope.instance}" class="${requestScope.descriptor.clazz}" page="config.jelly" optional="true"/>
```